File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/hyperlinks/UpdateHyperlinkWorksheet.java
package com.aspose.cells.cloud.examples.hyperlinks;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class UpdateHyperlinkWorksheet {

    public static void main(String... args) throws IOException {
        // ExStart: update-hyperlink-worksheet
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "sample1.xlsx";
            Path inputFile = Utils.getPath(UpdateHyperlinkWorksheet.class, input);
            String output = "sample2.xlsx";
            Path outputFile = Utils.getPath(UpdateHyperlinkWorksheet.class, output);

            String sheetName = "Sheet1";
            Integer hyperlinkIndex = 0;
            com.aspose.cells.model.Hyperlink body = new com.aspose.cells.model.Hyperlink();
            body.setAddress("http://www.aspose.com/cloud/total-api.aspx");
            body.setTextToDisplay("Aspose Cloud APIs");

            storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

            cellsApi.PostWorkSheetHyperlink(input, sheetName, hyperlinkIndex, Utils.STORAGE, Utils.FOLDER, body);

            com.aspose.storage.model.ResponseMessage sr = storageApi.GetDownload(input, null, Utils.STORAGE);

            Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: update-hyperlink-worksheet
    }

}

File: Examples/DotNet/CSharp/hyperlinks/UpdateHyperlinkWorksheet.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Hyperlinks
{
    class UpdateHyperlinkWorksheet
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "Sample_Test_Book.xls";
            String sheetName = "Sheet2";
            int hyperlinkIndex = 0;
            String storage = "";
            String folder = "";

            Hyperlink body = new Hyperlink();
            body.Address = "http://www.aspose.com/cloud/total-api.aspx";
            body.TextToDisplay = "Aspose Cloud APIs";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to update hyperlinks
                HyperlinkResponse apiResponse = cellsApi.PostWorkSheetHyperlink(fileName, sheetName, hyperlinkIndex, storage, folder, body);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine("Hyperlink Address : " + apiResponse.Hyperlink.Address);
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
